js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/ExecuteChecksButton.tsx

```diff
@@ -1,24 +1,24 @@
+import {gql} from '@apollo/client';
 import {Button, Icon, Spinner, Tooltip} from '@dagster-io/ui-components';
 import React, {useState} from 'react';
 
 import {usePermissionsForLocation} from '../../app/Permissions';
-import {AssetKeyInput} from '../../graphql/types';
+import {AssetCheckCanExecuteIndividually} from '../../graphql/types';
 import {useLaunchPadHooks} from '../../launchpad/LaunchpadHooksContext';
 
-export type EvaluateChecksAssetNode = {
-  assetKey: AssetKeyInput;
-  repository: {name: string; location: {name: string}};
-  jobNames: string[];
-};
```

Comment:
A lot of the GraphQL changes in this PR are replacing this with a proper fragment!